src/main/scala/codacy/dockerApi/DockerEngine.scala

```diff
@@ -61,10 +61,12 @@ abstract class DockerEngine(Tool: codacy.docker.api.Tool) extends Delayed {
         Tool.apply(
           source = Source.Directory(sourcePath.toString()),
           configuration = patternsOpt,
-          files = filesOpt
+          files = filesOpt,
+          options = configOpt.flatMap(_.options).getOrElse(Map.empty)
         )(spec)
       } catch {
         case t: Throwable =>
+          t.printStackTrace(Console.err)
```

Comment:
But the value is being returned. The person receiving it should decide. Here does not seem a good place to print values without option to remove verbosity.